### DISCUSSION TITLE
Github issue template-

### DISCUSSION THREAD

**Discussion by shivarm** [Hyperfoil/Horreum#466] [Ideas]
Why comment the fields out ?

We don't need to show the information in visual form to the contributors, all this can also be shown by commenting the field when the contributors open issues he will be able to see all info. This way we can make a better look of the issue template.

**Comment by jesperpedersen** [comment]
I don't agree.

Making the fields explicit highlights to users that they are mandatory to fill out.

**Reply by shivarm** [reply]
okay @jesperpedersen, np
